src/database/createAdmin.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable no-console */
+import { config } from 'dotenv';
+import roleService from '../services/role';
+import models from './models';
+
+const { User } = models;
+config();
+
+const createAdmin = async () => {
+  const roleName = 'SUPER_ADMIN';
+
+  const role = await roleService.findRoleByName(roleName);
+  if (!role) {
+    console.log(`FATAL: create ${roleName} role first.[Hint: run 'npm run role:create' to create role]`);
+    return;
+  }
+  const hashedPswd = process.env.ADMIN_PASSWORD;
+
+  const admin = {
+    firstname: 'Nomad',
+    lastname: 'admin',
+    email: 'spacenomad@gmail.com',
+    roleId: role.id,
+    isVerified: true,
+    password: hashedPswd,
```

Comment:
Can you add a comment above this line to guide people about this hashedpassword (**ADMIN_PASSWORD** used in .env) because it is sophisticated to reveal?
Or you highlight that in the PR description.

Reply:
Check PR descriptions

Reply:
I think comment is not needed that much since I stored value in constant which is being read from `.env` which is self explanatory